docs/docsite/Makefile

```diff
@@ -11,6 +11,11 @@ else
 CPUS ?= $(shell nproc)
 endif
 
+# Sets the build output directory if it's not already specified
+ifndef BUILDDIR
+	BUILDDIR = _build
+endif
+
```

Comment:
I think '?=' should work here.

(The `ifdef MODULES` etc cases are needed because the make var being set is a different name
than the env/cli var. ie, 'MODULES=foo make' changes MODULE_ARGS)


``` makefile
BUILDIR ?= _build
```